nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -149,3 +221,64 @@ def get_statistic_points(self, model: TModel, graph: NNCFGraph) -> StatisticPoin
         :param graph: Model graph.
         :return: Statistic points, for which StatisticsCollector should collect statistics.
         """
+
+    def _get_activations(
+        self, dataset: Dataset, nodes_to_compress: List[NNCFNode], graph: NNCFGraph, model: TModel
+    ) -> Dict[str, List[np.ndarray]]:
+        """
+        Collects input activations for the given nodes on the dataset.
+
+        :param dataset: Dataset to collect values.
+        :param nodes_to_compress: List of nodes, whose inputs are collected.
+        :param model: Model for statistics collection.
+        :param graph: Model graph.
+        :return: statistics values itself per node name.
+        """
+        subset_size = 128
+
+        activations = {}
+        _collected_stat_inputs_map = {}
+        statistic_container = StatisticPointsContainer()
+        all_act_nodes = []
+        act_vs_shared_node_names_mapping = defaultdict(list)
+        from nncf.openvino.graph.metatypes.openvino_metatypes import OVMatMulMetatype
```

Comment:
import should be placed at the top of the file

Reply:
@ljaljushkin, please consider to move this property (MM metatype) to the backend method

Reply:
done